src/scitokens.h

```diff
@@ -295,21 +295,47 @@ int keycache_set_jwks(const char *issuer, const char *jwks, char **err_msg);
  */
 
 /**
- * Update scitokens parameters.
+ * Update scitokens int parameters.
  * Takes in key/value pairs and assigns the input value to whatever
  * configuration variable is indicated by the key.
  * Returns 0 on success, and non-zero for invalid keys or values.
  */
 int config_set_int(const char *key, int value, char **err_msg);
 
+// Prefixed version of the same API to avoid potential symbol collisions
+int scitokens_config_set_int(const char *key, int value, char **err_msg);
+
 /**
- * Get current scitokens parameters.
+ * Get current scitokens int parameters.
  * Returns the value associated with the supplied input key on success, and -1
  * on failure This assumes there are no keys for which a negative return value
  * is permissible.
  */
 int config_get_int(const char *key, char **err_msg);
 
+// Prefixed version of the same API to avoid potential symbol collisions
+int scitokens_config_get_int(const char *key, char **err_msg);
+
+/**
+ * Set current scitokens str parameters.
+ * Returns 0 on success, nonzero on failure
+ */
+int config_set_str(const char *key, const char *value, char **err_msg);
```

Comment:
If config_get/set_int() will be deprecated, shouldn't we not be introducing config_get/set_str()?